frontend/src/pages/organization/NetworkingPage/components/RelayTab/components/RelayDeployModal.tsx

```diff
@@ -4,14 +4,20 @@ import { Modal, ModalContent } from "@app/components/v2";
 import { RelayDeploymentMethodSelect } from "@app/pages/organization/NetworkingPage/components/RelayTab/components/RelayDeploymentMethodSelect";
 
 import { RelayCliDeploymentMethod } from "./RelayCliDeploymentMethod";
+import { RelayTerraformDeploymentMethod } from "./RelayTerraformDeploymentMethod";
 
 type Props = {
   isOpen: boolean;
   onOpenChange: (isOpen: boolean) => void;
 };
 
 export const RelayDeploymentInfoMap = {
-  cli: { name: "CLI", image: "SSH.png", component: RelayCliDeploymentMethod }
+  cli: { name: "CLI", image: "SSH.png", component: RelayCliDeploymentMethod },
```

Comment:
IGNORE PLACEMENT OF COMMENT

I think we can hide this host address field for both CLI and terraform options... if they'd want it to point to a different URL, they'd be on that Infisical platform 
<img width="847" height="600" alt="image" src="https://github.com/user-attachments/assets/7fc32ae1-0d86-40d2-af6e-a8e2a9778c0a" />